src/trackers/helpers/fingerprints.js

```diff
@@ -0,0 +1,55 @@
+// Bucket fingerprint scores into a 0-3 scale.
+// The cutoffs for each bucket were determined manually by looking
+// at previous crawl data.
+function getFingerprintRank (fp) {
+    if (!fp) {return 0}
+
+    const logFp = Math.log(fp)
+    let bucket = 0
+
+    if (logFp > 6) {
+        bucket = 3
+    } else if (logFp > 3.5 && logFp <= 6) {
+        bucket = 2
+    } else if (logFp <= 3.5) {
```

Comment:
those magic values, similarly to api threshold, is something that we should probably also update automatically in the long run, right?

Reply:
Yes eventually